test/membrane_http_adaptive_stream/bandwidth_calculator_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Membrane.HTTPAdaptiveStream.BandwidthCalculatorTest do
+  use ExUnit.Case
+  use Ratio, comparison: true
+
+  alias Membrane.HTTPAdaptiveStream.Manifest.Track
+  alias Membrane.HTTPAdaptiveStream.BandwidthCalculator
+  alias Membrane.Time
+
+  describe "Test bandwidth calculation for track with segment subsequences with duration " do
+    test "equal to zero" do
+      # All duration are 0 so bandwidth calculation is impossible.
+      # Therefore high but realistic default bandwidth is returned.
+      test_track = mock_track([{0, 0.0}, {1, 0.0}], 5)
+      assert BandwidthCalculator.calculate_bandwidth(test_track) == 2_560_000
+    end
+
+    test "out of 0.5 - 1.5 of target bandwidth scope" do
+      # All segment subsequences have duration out of 0.5 - 1.5 target duration range.
+      # Therefore this condition is omitted.
```

Comment:
nit: "this condition"? I would consider removing sentences starting from "Therefore". Unit test should revolve around only input and output values and here you somehow details of internals on the module.